db/seeds.rb

```diff
@@ -10,3 +10,31 @@
 Category.create(label: 'self_care')
 Category.create(label: 'news')
 Category.create(label: 'culture')
```

Comment:
are these the categories that you have in client? remember that you have the labels of the categories hardcoded there.

Reply:
yes and then we are sending category_id from client :)

Reply:
@ViktorHek yes but in your client you don't have self_care, news and culture but rather global_politics, sports, self-care, news and culture. So if i in the client press on global politics i will  with these seeds instead get articles about self care, and in your client your have more categories than it exists in your database after this, which will lead to an error, that you don't have an error handler for in your controller